dags/openshift_nightlies/tasks/install/openstack/shift_on_stack.py

```diff
@@ -0,0 +1,30 @@
+import sys
+from os.path import abspath, dirname
+from os import environ
+
+sys.path.insert(0, dirname(dirname(dirname(abspath(dirname(__file__))))))
+from util import var_loader, kubeconfig, constants
+from tasks.install.openshift import AbstractOpenshiftInstaller
+
+import json
+import requests
+
+from airflow.operators.bash_operator import BashOperator
+from airflow.models import Variable
+from kubernetes.client import models as k8s
+
+# Defines Tasks for installation of Openshift Clusters
+class ShiftOnStackInstaller(AbstractOpenshiftInstaller):
```

Comment:
Can this be called either `openstack.py`/`OpenstackInstaller` or `jetpack.py`/`OpenstackJetpackInstaller`  to make it easier to map the classes to what platform/installer they are responsible for

Reply:
fixed